docs/tutorials/02a_training_a_quantum_model_on_a_real_dataset.ipynb

```diff
@@ -201,7 +201,7 @@
     },
```

Comment:
Wonder why figures in this tutorial were updated.

Reply:
That's an accidental inclusion. I'll remove those changes.